projects/swimlane/ngx-ui/src/lib/services/icon-registry/convert-class.util.ts

```diff
@@ -1,4 +1,4 @@
-export function convertClass(input: string): string {
+export const convertClass = (input: string): string => {
```

Comment:
thoughts: I've seen a couple of instances of converting function over to arrow function. Are we going to make this our convention?

Reply:
Yes... we prefer arrow to function.   Perhaps a warning instead of an error?